script.js

```diff
@@ -71,10 +73,15 @@ function checkWinner() {
     ) {
       winMessage = "Player 2 wins!";
     }
+    if (winMessage == '' && turns === 9){
```

Comment:
Suggest using === when checking the winMessage as well to avoid any weirdness with type juggling

Reply:
Done